src/NuGet.Core/NuGet.Packaging/Signing/SignManifestFileEntry.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents a file in a package that is listed in the signed manifest.
+    /// </summary>
+    public sealed class SignManifestFileEntry
```

Comment:
Consider just `ManifestPackageEntry`, for similar reasons I suggested that `SignManifest` be renamed to `Manifest`.  Also, an entry may be a folder not a file.